docs/conf.py

```diff
@@ -48,7 +48,13 @@ def _check_objects_are_documented():
                 raise RuntimeError(f"World object {name} is not documented.")
 
 
-_check_objects_are_documented()
+# TODO: @almarklein it is a bit odd to have a test like this in the config file
+# that will build the docs in particular because it runs before the docs are
+# built ... could we make it a "normal" unit test in something like `test_docs`?
+# Also: this breaks when we autogenerate the docs, because there will be no
+# explicit `.. autoclass::` line anymore. I could work out how to adapt this
+# function though, if you think it is worth keeping.
+# _check_objects_are_documented()
```

Comment:
You still have to manually "mention" a new function in the autosummary list of the respective module. It's harder to miss, because you will see the module's docstring when you add the function to the module's `__init__.py`, but one could still miss it. (We could automate this, too, but it significantly reduces our flexibility in writing nice docs.)

That said, I'm not aware of any other repo that has such a check, so keeping/discarding depends on how "bad" we think it is if we miss a function in the docs. (We can always add it later and the docs update instantly.)

Reply:
Anyway, let's drop the check here.